thirdeye-ui/src/app/routers/subscription-groups/subscription-groups-create-edit.router.interface.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2022 StarTree Inc
```

Comment:
nit. do you want to change this to 2023 as well?

Reply:
updated